docs/how_to_guides/ingest_data/Derivation Strategy.md

```diff
@@ -0,0 +1,302 @@
+---
+id: derivation_strategies
+title: Schema Derivation Strategies
+
+---
+
+* Guzzle provides five schema derivation strategies. Using these strategies Guzzle will validate the data type and will also create an effective schema based on that data will be written in the target and reject table. The schema derivation strategies include:
+
+    1. Source
+    2. Inherit source data type and do not apply validation
+    3. Target
+    4. Inherit target data type and do not apply validation
+    5. Do not inherit schema from source or target
+
+* Users can select schema derivation strategy from **Inherit columns and datatype** property under **Additional settings** dropdown menu in the validation and transformation section.
+
+* In this article, we will discuss these strategies with examples
+
+## Source
+
+* If schema derivation strategy is **source** then, columns and data types will be inherited from the source files/table and, validation of data types will be performed based on source data types.
+* If the user does not specify the column name in the transformation and validation section then validation for that column will be performed and if the user specifies the source's column name in the transformation and validation section then the validation will be performed based on validate data type flag.
+* Additionally, an effective schema will be created using source columns and data types and columns and data types specified in the **validation and transformation** section of the ingestion job. And this effective schema will be used to create a table if an auto-create table flag is true in the target section and in the reject section.
+
+### Example
+
+* In source, a datastore is a file and format is delimited, and the file pattern includes two files that are file1.csv and file2.csv. And infer schema flag is true. In target, the datastore is the delta table and the auto-create table flag is true and the name of table is **demo_schema_derivation**.
+
+* The source data are like this 
+```
+file1.csv
+
+id,name
+1001,n1
+1002,n2
+```
+```
+file2.csv
+
+id,name
+1003,n3
+id_1004,n4
+```
+* In the validation and transformation section, we have defined 
+1. A new column **full_name** is doing concatenation of **name** column and "_new" string. If we do not provide any data type then Guzzle will take a string as the data type for this column and if the data type field has a value then it will take that data type for that column. As you can see from the below image.
+2. **name** column but a validate data type flag is **false**, and also we haven't defined any data type so, Guzzle will not validate this column's data and the data type for this column will be inherited from the source as the name column is present in the source.
```

Comment:
can be more clear, the reason of guzzle will not validate the data type is due to `validate data type checkbox is unchecked` not the we haven't define any data type, so what about this ?

- **name** column
  - situation 
     - this column does exists in source
     - the validate data type checkbox is unchecked 
     - data type text box empty
  - result
     - guzzle will respect the `validate data type checkbox = unchecked` and will not do any data type validation
     - since column exists in source then data type will be inherited from source